jest.config.mjs

```diff
@@ -13,6 +13,9 @@ export default {
     '^.+\\.(t|j)sx?$': 'ts-jest'
   },
   testRegex: '(/tests/.*|\\.(test))\\.(ts|tsx|js)$',
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1'
+  },
```

Comment:
I had to add this because files imported with the `.js` extension weren't seen by jest, as jest still have a limited support for ESM. I followed `ts-jest`: https://kulshekhar.github.io/ts-jest/docs/guides/esm-support/#examples

Reply:
Just a small 2 cents on the Jest ESM interop for the maintainers of TinyHTTP:

Jest caused me all kinds of headaches with ESM. With [PlopJS](https://github.com/plopjs/plop) we migrated everything to [Vitest](https://vitest.dev/) in ~5 minutes (it has a Jest compat mode, so no need to rewrite tests) and have seemingly had no related issues that we did with Jest.

Happy to send a follow-up PR if this sounds interesting to the TinyHTTP maintainers.

Reply:
I'd be down